versioned/mongodb/src/test/java/com/dremio/nessie/versioned/store/mongodb/SampleEntities.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.versioned.store.mongodb;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.dremio.nessie.versioned.impl.Fragment;
+import com.dremio.nessie.versioned.impl.InternalCommitMetadata;
+import com.dremio.nessie.versioned.impl.InternalRef;
+import com.dremio.nessie.versioned.impl.InternalValue;
+import com.dremio.nessie.versioned.impl.L1;
+import com.dremio.nessie.versioned.impl.L2;
+import com.dremio.nessie.versioned.impl.L3;
+import com.dremio.nessie.versioned.store.Entity;
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * This utility class generates sample objects mapping to each enumerate in
+ * {@link com.dremio.nessie.versioned.store.ValueType}.
+ * It is intended to be a central place for test data generation used by multiple test suites.
+ */
+public class SampleEntities {
+  private static final Entity ID = Entity.ofBinary(
+      new byte[] {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19});
```

Comment:
let's not use hand written arbitrary byte arrays. If you want something arbitrary, please declare using a Random. If you want a consistent random, use Random with a specified seed.